packages/indexer-agent/src/network.ts

```diff
@@ -523,9 +523,11 @@ export class Network {
         const results = result.data.subgraphDeployments
 
         // In the case of a fresh graph network there will be no published subgraphs, handle gracefully
-        if (results.length == 0) {
-          this.logger.warn('No subgraph deployments returned')
-          return []
```

Comment:
This was returning an empty `targetDeployments` array if the last batch query happened to be empty. 🤦

Reply:
ahhh :0!!!!

for fresh graph networks, it should still give the warning below? Could there be one that checks for 'freshness'?